src/app/providers/config/ServiceConstants.ts

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
Will move that to UI-Shared if we agree? Those values aren't used anywhere else yet.

Reply:
I think it's ok to keep it here. The shared repo is troublesome because of the way dependencies are resolved by webpack at runtime for federated modules, we have had occasions with mismatched providers and consumers and a broken app. With @pmuir we talked about passing the infos as props to the federated modules, but it's an extensive change and we are prioritizing other parts right now.

We can always move this later if needed.